validation/src/test/scala/hmda/validation/rules/lar/validity/V215Spec.scala

```diff
@@ -0,0 +1,38 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V215Spec extends LarEditCheckSpec {
+  property("If action taken type must = 6, then date application received = NA") {
```

Comment:
They way I have been doing it in my tests, I just test that it passes when application date == NA. I believe that it gets slightly higher test coverage since the property tests will also cover when action taken == 6 and not matter when the action taken is the test should pass.